simple_bank_online/lib/simple_bank_online/router.ex

```diff
@@ -0,0 +1,39 @@
+defmodule SimpleBankOnline.Router do
+  use Plug.Router
+
+  plug :match
+  plug Plug.RequestId
+  plug Plug.Logger, log: :info
+  plug Plug.Parsers, parsers: [:json], pass: ["text/*"], json_decoder: Poison
+  plug :dispatch
+
+  get "/", do: send_resp(conn, 200, Poison.encode!(%{msg: "Welcome to Simple Bank Online!"}))
+
+  post "/accounts" do
+  end
+
+  get "/accounts/:id" do
+  end
+
+  post "/accounts/:id/deposits" do
+  end
+
+  post "/accounts/:id/withdrawals" do
```

Comment:
It really wouldn't surprise me if this was how my bank had it in their account management system